tensorflow/compiler/tests/build_defs.bzl

```diff
@@ -85,10 +86,16 @@ def tf_xla_py_test(
                 "--types=DT_HALF,DT_FLOAT,DT_DOUBLE,DT_UINT8,DT_QUINT8,DT_INT8,DT_QINT8,DT_INT32,DT_QINT32,DT_INT64,DT_BOOL,DT_COMPLEX64,DT_COMPLEX128",
             ]
         elif backend == "gpu":
-            backend_args += [
-                "--test_device=" + gpu_xla_device,
-                "--types=DT_HALF,DT_FLOAT,DT_DOUBLE,DT_UINT8,DT_QUINT8,DT_INT8,DT_QINT8,DT_INT32,DT_QINT32,DT_INT64,DT_BOOL,DT_COMPLEX64,DT_COMPLEX128,DT_BFLOAT16",
-            ]
+            if rocm_is_configured():
+                backend_args += [
+                    "--test_device=" + gpu_xla_device,
+                    "--types=DT_HALF,DT_FLOAT,DT_DOUBLE,DT_UINT8,DT_QUINT8,DT_INT8,DT_QINT8,DT_INT32,DT_QINT32,DT_INT64,DT_BOOL,DT_COMPLEX64,DT_BFLOAT16",
+                ]
+            else:
+                backend_args += [
```

Comment:
Is it possible to avoid the duplication in these cases? Currently it's hard to see what is the difference between the cases.

Reply:
good point.

I will make this change and push out the commit

Reply:
removed the special casing for rocm altogether. since the PR to switch to ROCm 3.0 was recently merged, this is no longer needed :)